src/Microsoft.HttpRepl/Commands/PrefCommand.cs

```diff
@@ -257,5 +271,24 @@ protected override IEnumerable<string> GetArgumentSuggestionsForText(IShellState
 
             return null;
         }
+
+        private static bool IsVSCode(string path)
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                return path.Contains("Code.exe", StringComparison.OrdinalIgnoreCase) ||
+                       path.Contains("Code - Insiders.exe", StringComparison.OrdinalIgnoreCase);
+            }
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+            {
+                return path.Contains("Visual Studio Code.app", StringComparison.Ordinal) ||
+                       path.Contains("Visual Studio Code - Insiders.app", StringComparison.Ordinal);
+            }
+            else //  Linux
+            {
+                return string.Equals(path, "/usr/bin/code", StringComparison.Ordinal) ||
```

Comment:
Doing a full path equals seems like we could miss cases.  What if it's in `~/bin` or `/usr/local/bin` or some variant?  (I don't know if anyone would do this, but since we only check executable name on the other two platforms, this one stands out...)

Reply:
I went back and forth on this. I was worried about just doing, essentially, `.Contains("code")` thinking it might catch false positives. The others OSs have more explicit things we can check for. I guess maybe it could be `.EndsWith("code")` and that might have fewer false positives? I think we're already talking about a small number of cases... just not sure which is the right slightly smaller number of cases :)

Reply:
Or an `.EndsWith("/bin/code")` might catch a few more, without pinning solely to the /usr mount point.  But it is trying to catch a small handful of edge cases, so feel free to punt on it too. 😉

Reply:
I expect that this will evolve over time as we get feedback from users, so I think we can start here and see what happens. It might need to change even further if we ever change the tool so the whole path isn't required (so the command could just be "code" on basically all platforms and it'd work). Right now that doesn't work because of how we use `ProcessStartInfo`/`Process.Start` (with `ShellExecute` false being the issue I believe) but I _think_ there are some options there. Just hasn't been a priority yet.